contracts/ERC20.sol

```diff
@@ -209,8 +209,8 @@ contract ERC20 is IERC20 {
 
         _beforeTokenTransfer(account, address(0), amount);
```

Comment:
Let's remove this method now? No need to have newly deployed tokens waste gas with this